setup.cfg

```diff
@@ -15,3 +15,8 @@ omit =
 	*/backends*
 	node_modules/*
 
+[pycodestyle]
+# error codes: https://pycodestyle.readthedocs.io/en/latest/intro.html#error-codes
+# E501: line too long
+ignore=E501
```

Comment:
Even you are ignoring line too long, I think there should be a number defined as 120. @diegomillan 
max-line-length = 120

Reply:
But it will be ignored @Squirrel18. We should add it if we decide to remove E501